### PR TITLE
chore(Makefile): reduce ui-tests build and runtime

### PR DESCRIPTION
**Pull request by caybro**
### What does the PR do

- do not depend on NIM for building/running StatusQ and Storybook stuff

### Affected areas

Makefile/build

### CODE REVIEW

**Comment by caybro** [comment]
:partying_face: it's roughly ~2 minutes faster than the average of the past 20 runs

File: Makefile [hunk 1]
```diff
@@ -149,7 +149,8 @@ $(BOTTLES_DIR):
 bottles: $(BOTTLES)
 endif
 
-deps: | check-qt-dir deps-common bottles
+non-nim-deps: check-qt-dir bottles
```
Comment (micieslak) [inline Makefile:152]: I would ensure if `bottles` is nim or non-nim dependency. At first glance it's for ssl libraries on mac, required for nim part. If it's like that non-nim-deps would be only check-qt-dir.

Comment (osmaczko) [reply Makefile:152]: If that's the case, then I believe we don't need `non-nim-deps`, the name sounds a bit artificial.

Comment (caybro) [reply Makefile:152]: Right, any idea @alexjba ?

Comment (caybro) [reply Makefile:152]: or @igor-sirotin

Comment (alexjba) [reply Makefile:152]: We're using it for Qt. In fact both Qt and our nim code have a dependency on openssl (or had at some point). But this solution is quite old and from what I see in the PR is meant to support qt5.14 on really old macOS versions. Maybe it's worth revisiting it and at least test if we still need this 😄 
https://github.com/status-im/status-desktop/pull/375

Comment (caybro) [reply Makefile:152]: Yeah but this doesn't really matter in the context of `ui-tests` as those are run on Linux only

Comment (alexjba) [reply Makefile:152]: But what's for sure is that it's a nim dependency. We do a static link in `configs.nim`

Comment (caybro) [reply Makefile:152]: Alright, I can remove it and just make the SQ/SB stuff depend on `check-qt-dir`

Comment (caybro) [reply Makefile]: Done

**Review by jakubgs — APPROVED** [review]
Nice.

**Review by alaibe — APPROVED** [review]

**Pull request merged**